local-setup/launch.py

```diff
@@ -215,24 +216,38 @@ def get_subcommand_flags(index):
         "--request-state" if index > 0 else None
     ]))
 
-def main(processes, workers_number, parachain_type, log_config_path, offset, parachain_dir):
+def main(processes, worker, workers_number, parachain_type, log_config_path, offset, parachain_dir):
     # Litentry
+    if worker == "identity":
+        worker_dir = "tee-worker"
+    elif worker == "bitacross":
+        worker_dir = "bitacross-worker"
+    else:
+        sys.exit("Unsupported worker")
+
+    if worker == "identity":
+        worker_bin = "litentry-worker"
+    elif worker == "bitacross":
+        worker_bin = "bitacross-worker"
+    else:
+        sys.exit("Unsupported worker")
```

Comment:
shall we just smush it together?

```suggestion
    if worker == "identity":
        worker_dir = "tee-worker"
        worker_bin = "litentry-worker"
    elif worker == "bitacross":
        worker_dir = "bitacross-worker"
        worker_bin = "bitacross-worker"
    else:
        sys.exit("Unsupported worker")
```